### DISCUSSION TITLE
One JCB Version

### DISCUSSION THREAD

**Discussion by Llewellynvdm** [joomengine/Joomla-Component-Builder#927] [Announcements]
### We will move all the features JCB has (including Pro Member features) public.

While we have been working very hard to grow the JCB Pro contribution, it has become clear to us that instead of increasing the growth of JCB this way, it has slowed it down.

So we had two options, drop support for the public version and make JCB Pro the only version we directly support, or drop the JCB Pro and make the public version the only version we directly support.

Since dropping the Public version will break our commitment to freedom software and, more importantly, destroy any ground of trust we have build with you. That was clearly not an option, and as result... we are ending the PRO Member Version of JCB.

### Where does this leave the Pro Members?

At our last [JCB JUG](https://t.me/Joomla_component_builder) meeting we asked those Pro Members that where in attendance what they think, and these where some of the responses we got:

- We would continue to support JCB.
- We hardly used the Pro Features as they are not easy to find.
- What exactly were the Pro Features?
- My Pro Subscription was never about getting more, but giving back.
- Thank you so much for this tool, we trust you will make the best choice that will, as always, benefit all using JCB

This was a huge encouragement to me as the lead developer and founder of JCB. The current idea is that all Pro Members will retain their status as Pro Members, and will have voting rights on selected topics, access to exclusive JCB content, and other non-public information.

We will therefore continue to encourage JCB users to become JCB Pro Members if for no other reason but to make sure that JCB has the needed funding and support it needs to grow, and adapt in a world of constant change.

Let the celebrations begin as we move to a single version that is completely public.

**Comment by Schrijvers123** [comment]
As promemebr status we made donations the Pro version. Is that converted now also to the version 4 project?

**Reply by Llewellynvdm** [reply]
Any Pro Member contribution will still as before go towards developing JCB. Only change is that we will no longer support two versions a **public** and a **pro**... we will **now** just support the **public** version.

Our commitment to move to Joomla 4 remains the same... we are working hard towards this and in fact this change will increase the progress, as the powers area (that use to only be available to Pro) will now go public and with it we can move so much faster to Joomla 4.
